### PR TITLE
Release 0.2.1

### PR DESCRIPTION
**Pull request by mattwthompson**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by mattwthompson** [comment]
@conda-forge-admin, please rerender

**Comment by mattwthompson** [comment]
This freaking error is so annoying, I have literally only ever seen it triggered by `conda-build`

```
import: 'openff.fragmenter'
Traceback (most recent call last):
  File "/home/conda/feedstock_root/build_artifacts/openff-fragmenter-split_1691438569832/test_tmp/run_test.py", line 2, in <module>
    import openff.fragmenter
  File "/home/conda/feedstock_root/build_artifacts/openff-fragmenter-split_1691438569832/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pl/lib/python3.10/site-packages/openff/fragmenter/__init__.py", line 7, in <module>
    from . import chemi, fragment, utils
  File "/home/conda/feedstock_root/build_artifacts/openff-fragmenter-split_1691438569832/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pl/lib/python3.10/site-packages/openff/fragmenter/chemi.py", line 8, in <module>
    from openff.fragmenter.utils import get_atom_index, get_map_index
  File "/home/conda/feedstock_root/build_artifacts/openff-fragmenter-split_1691438569832/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pl/lib/python3.10/site-packages/openff/fragmenter/utils.py", line 6, in <module>
    from openff.toolkit.topology import Molecule
  File "/home/conda/feedstock_root/build_artifacts/openff-fragmenter-split_1691438569832/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pl/lib/python3.10/site-packages/openff/toolkit/topology/__init__.py", line 1, in <module>
    from openff.toolkit.topology.molecule import (
  File "/home/conda/feedstock_root/build_artifacts/openff-fragmenter-split_1691438569832/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pl/lib/python3.10/site-packages/openff/toolkit/topology/molecule.py", line 57, in <module>
    from openff.toolkit.utils.constants import DEFAULT_AROMATICITY_MODEL
  File "/home/conda/feedstock_root/build_artifacts/openff-fragmenter-split_1691438569832/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pl/lib/python3.10/site-packages/openff/toolkit/utils/__init__.py", line 9, in <module>
    from openff.toolkit.utils.toolkits import (
  File "/home/conda/feedstock_root/build_artifacts/openff-fragmenter-split_1691438569832/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pl/lib/python3.10/site-packages/openff/toolkit/utils/toolkits.py", line 134, in <module>
    from openff.toolkit.utils import all_subclasses
ImportError: cannot import name 'all_subclasses' from partially initialized module 'openff.toolkit.utils' (most likely due to a circular import) (/home/conda/feedstock_root/build_artifacts/openff-fragmenter-split_1691438569832/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pl/lib/python3.10/site-packages/openff/toolkit/utils/__init__.py)
Tests failed for openff-fragmenter-base-0.2.1-pyhd8ed1ab_0.conda - moving package to /home/conda/feedstock_root/build_artifacts/broken

**Comment by mattwthompson** [comment]
This has never been an issue in our more realistic testing infrastructure, has never been an issue in version 0.2.0, and nothing in the [diff](https://github.com/openforcefield/openff-fragmenter/compare/0.2.0...0.2.1) implies to me that a change in the code caused this. I'm just going to drop the import test.

**Comment by mattwthompson** [comment]
@conda-forge-admin, please rerender

**Comment by mattwthompson** [comment]
ugh ...

**Comment by j-wags** [comment]
@lilyminium identified the root cause and suggested [a patch](https://github.com/openforcefield/openff-toolkit/pull/1690) which I plan to merge tomorrow. In the short term, I think you may be able to get past this by adding `rdkit` to the `host` requirements. And if that doesn't work I'll put out an OFFTK release with that PR tomorrow morning.

**Comment by j-wags** [comment]
Well, darn. I'll cut 0.14.3 with the fix for this tomorrow morning then!

File: recipe/meta.yaml [hunk 1]
```diff
@@ -20,34 +20,35 @@ outputs:
...
+        - rdkit
       run:
-        - python >=3.8
+        - python >=3.9
         - openff-toolkit-base >=0.11.0
         - numpy
-        - pydantic <2.0.0a0
+        - pydantic <3.0.0a0
         - networkx
         - jinja2
```
Comment (lilyminium) [inline recipe/meta.yaml:31]:
```suggestion
        - jinja2
        - rdkit
```

Perhaps try it in `run` as well? It doesn't look like rdkit is installed in the test environment.

**Comment by mattwthompson** [comment]
I'm inclined to keep `rdkit` out of the base package - IIUC the split is mostly to keep things from falling back to `sqm` but I'd still like to make as small a set of changes as possible. The diff between 0.2.0 (almost a year ago) and this release is quite small, so I'm inclined to believe that it's not a big deal if either

1. if we have to wait a couple days to roll out 0.14.3
2. if a solver drops back to 0.2.0

**Comment by mattwthompson** [comment]
Thanks for the rapid turnaround - everything looks good here

**Pull request merged**